templates/README.md

```diff
@@ -22,6 +22,17 @@ To set up your local environment with access to AWS and Kubernetes, just run:
 ```
 This script will open a web browser and prompt you to log in with your Commit Gmail account, and then will configure an AWS profile and a Kubernetes context.
 
+# Creating secrets
+
+The Playground uses a somewhat "quick and dirty" way to create secrets for your application without needing to commit them to your GitHub repository. In order to add secrets to your deployment:
+
+1. Rename `secrets/rename-these-secrets.yml` to `secrets.yml` (note that `secrets.yml` has been added to the `.gitignore` file, so they will not be committed to your GitHub repository).
```

Comment:
Hopefully these instructions should be clear

Reply:
looks great! 

maybe rename the file to something like `secrets.yml.example` or `example-secrets.yml` ? I think the instruction is clear enough?

Reply:
Sure!